src/fides/ctl/core/deploy.py

```diff
@@ -65,10 +65,19 @@ def compare_semvers(version_a: List[int], version_b: List[int]) -> bool:
 
 def check_docker_version() -> bool:
     """Verify the Docker version."""
+    try:
+        run("docker info", stdout=PIPE, stderr=PIPE, check=True, shell=True)
```

Comment:
The output is not currently seen, but as written, changes to the default error handling could unexpectedly print output from this command. Since we're catching the exception and printing our own error, we can explicitly swallow this output:
```py
from subprocess import DEVNULL, STDOUT
```
then:
```suggestion
        run("docker info", stdout=DEVNULL, stderr=STDOUT, check=True, shell=True)
```

Reply:
Yeah- the intent is to consume error messages to avoid confusion 👍

Reply:
Solid suggestion, thanks!